internal/runners/packages/list.go

```diff
@@ -172,7 +218,18 @@ func newFilteredRequirementsTable(requirements []*gqlModel.Requirement, filter s
 
 		versionConstraint := req.VersionConstraint
 		if versionConstraint == "" {
-			versionConstraint = "Auto"
+			versionConstraint = locale.T("constraint_auto")
+			if len(req.VersionConstraints) > 0 {
+				reqs := model.MonoModelConstraintsToRequirements(&req.VersionConstraints)
+				versionConstraint = model.RequirementsToVersionString(reqs)
+			}
+
+			for _, a := range artifacts {
+				if a.Namespace == ns.String() && a.Name == req.Requirement {
```

Comment:
Should also check if versionConstrained doesn't already equal `*a.Version` like you did with languages.